bundles/core/src/test/resources/form/datepicker/exporter-datepicker.json

```diff
@@ -11,6 +11,10 @@
   "properties": {
     "af:layout" : {
       "tooltipVisible": false
+    },
+    "fd:dor": {
+      "dorExclusion":false,
+      "dorColspan":null
```

Comment:
Why are we including null. We should only include if non empty

Reply:
Incorporated